spec/integration/ruby_command_line_spec.rb

```diff
@@ -58,7 +58,7 @@ module SyntaxSuggest
         EOM
         script.write(contents)
 
-        out = `#{ruby} -I#{lib_dir} -rsyntax_suggest #{script} 2>&1`
+        out = `#{ruby} -I#{lib_dir} -rsyntax_suggest/version #{script} 2>&1`
```

Comment:
Can you give some context for why this change was made? Is it a performance issue or something else? 

I’m good with the PR overall but want to preserve some context so if someone refactors this later they will understand the intent.

Reply:
> `-rsyntax_suggest` didn't work with `make test-syntax-suggest` on ruby/ruby repository. It raises `uninitialized constant SyntaxSuggest::VERSION (NameError)`. I'm not sure why they are different result.

I believe because the lib by default doesn't require `version` file: https://github.com/ruby/syntax_suggest/blob/b57fe7f/lib/syntax_suggest.rb

I think it's OK practice. We can do in this way, or left it here and just `require_relative 'syntax_suggest/version'` in the main lib file.

Reply:
I think originally I did that as a kind of guard for if the code was loaded or not. But I’m using this now https://github.com/ruby/syntax_suggest/blob/b57fe7fc1d6b928c8f34cde0590e698350284da7/lib/syntax_suggest/core_ext.rb#L25

so if we want to load the version always I think it’s no problem

Reply:
I prefer to load the version file instead of autoload. 

I remember hitting some edgecase-y things with autoload when working on this. Unfortunately I didn’t write down exactly what those problems were.

Reply:
Yeah, I think simple `require_relative` with single `VERSION` constant would not make a big impact on performance.